### ISSUE TITLE
RadioButton: add `required`

### ISSUE THREAD

**Issue by Lukas742** [SAP/ui5-webcomponents#5896]
### **Feature Request Description**
For screen reader support we need to be able to mark every input-like component as required. For this we would need the `RadioButton` to accept the `required` property/attribute.

### **Proposed Solution**
Add `required` to the component's public properties/attributes.

### **Additional Context**
We need this for our [FilterBar](https://sap.github.io/ui5-webcomponents-react/?path=/docs/layouts-floorplans-filterbar--default-story) implementation. There it's possible to mark a filter as required and if a filter is required, it should also be read when focusing the input. 

### **Priority**
- [ ] Low
- [x] Medium
- [ ] High
- [ ] Very High

A clear and concise description of the impact/urgency of the required feature.

### **Stakeholder Info** (if applicable)

- Organization: UI5 Web Components for React

**Comment by gmkv** [comment]
Hi @SAP/ui5-webcomponents-topic-rd 
This is a feature request to add `required` to the RadioButton.
Would you have a look?

**Comment by TeodorTaushanov** [comment]
Hi @Lukas742,

To programmatically indicate that a radio button must be chosen from a radio group the aria-required attribute, with a value of true, must be specified on the radiogroup element. In your case you can use:
```
<div aria-labelledby="radioGroupTitle1" aria-required="true" role="radiogroup">
   <ui5-title id="radioGroupTitle1">Group1 - None pre-selection</ui5-title>
   <ui5-radio-button text="Radio1" name="test"></ui5-radio-button>
   <ui5-radio-button text="Radio2" name="test"></ui5-radio-button>
   <ui5-radio-button text="Radio3" name="test"></ui5-radio-button>
</div>
```

It is not expected to use the aria-required attribute on individual radio buttons.

Best,
Teo

**Comment by TeodorTaushanov** [comment]
Hi @Lukas742,

We implemented the "required" property to enable form validation, which will be available from version 1.9.0.

You still need to put the radio buttons inside an element with role="radiogroup" and  aria-required="true" to have the correct ARIA support:
```
<form>
  <div aria-labelledby="formRadioGroupTitle1" aria-required="true" role="radiogroup">
	  <ui5-title id="formRadioGroupTitle1">Select:</ui5-title>
	  <ui5-radio-button id="formRadioBtnRequired" value="optionA" name="formGroup" text="Option A" required></ui5-radio-button>
	  <ui5-radio-button value="optionB" name="formGroup" text="Option B"></ui5-radio-button>
	  <ui5-radio-button value="optionC" name="formGroup" text="Option C"></ui5-radio-button>
  </div>
  <input type="submit">
</form>
```

For the "required" property to have effect, you must add the following import to your project:
`import "@ui5/webcomponents/dist/features/InputElementsFormSupport.js";`


Best,
Teo
